src/nodule.js

```diff
@@ -0,0 +1,52 @@
+import getInjector from './injector';
+import {
+    loadEach,
+    loadFromCredstash,
+    loadFromEnvironment,
+    loadFromObject,
+} from './loaders';
+import Metadata from './metadata';
+
+
+export default class Nodule {
+
+    constructor(metadata, loaders = null, scope = null) {
```

Comment:
The default value for `scope` is actually 'nodule'.
So, the behavior of `Nodule(scope='nodule')` can be surprising for some.